q2_types/feature_table/_transformer.py

```diff
@@ -60,6 +60,12 @@ def _table_to_dataframe(table: biom.Table) -> pd.DataFrame:
     return pd.DataFrame(array, index=sample_ids, columns=feature_ids)
 
 
+def _table_to_metadata(table: biom.Table) -> qiime2.Metadata:
+    table = _table_to_dataframe(table)
+    table.index.name = '#SampleID'
```

Comment:
Can you make this `id` - that way it is applicable when we start working with transposed tables, too?